skimage/transform/_warps.py

```diff
@@ -390,6 +390,10 @@ def swirl(image, center=None, strength=1, radius=100, rotation=0,
         image is converted according to the conventions of `img_as_float`.
 
     """
+    if mode is None:
+        warn('The default of `mode` in `skimage.transform.warp` '
+             'will change to `reflect` in version 0.15.')
```

Comment:
@LachlanD as @soupault mentioned, you need to add the line "from warnings import warn" at the top of the file.